yopt/src/main/scala/pravda/yopt/CmdDecoder.scala

```diff
@@ -181,19 +177,20 @@ object CmdDecoder {
     override val optInfo: Option[String] = Some("<char>")
   }
 
-  // Sample values: 1 OR 1,2,3
   implicit def seqReader[A: CmdDecoder]: CmdDecoder[Seq[A]] = new CmdDecoder[Seq[A]] {
```

Comment:
That's not the best solution. Let's discuss how to pass several files to the CLI.

Reply:
@fomkin